lib/matplotlib/testing/exceptions.py

```diff
@@ -1,15 +1,3 @@
-class KnownFailureTest(Exception):
```

Comment:
By my logic - anything that is not a part of the public API can be changed or even removed at any time, so everyone who did something like using `KnownFailureTest` makes it on its own risk.